Documentation/PackageDescription.md

```diff
@@ -1056,35 +1056,48 @@ The supported C language standard to use for compiling C sources in the package.
 enum CLanguageStandard {
     case c89
     case c90
-    case iso9899_1990
-    case iso9899_199409
+    case c99
+    case c11
+    case c17
+    case c18
+    case c2x
     case gnu89
     case gnu90
-    case c99
-    case iso9899_1999
     case gnu99
-    case c11
-    case iso9899_2011
     case gnu11
+    case gnu17
+    case gnu18
+    case gnu2x
+    case iso9899_1990 = "iso9899:1990"
+    case iso9899_199409 = "iso9899:199409"
+    case iso9899_1999 = "iso9899:1999"
+    case iso9899_2011 = "iso9899:2011"
+    case iso9899_2017 = "iso9899:2017"
+    case iso9899_2018 = "iso9899:2018"
 }
 ```
+
 # CXXLanguageStandard
 
 `enum CXXLanguageStandard`
 
-The supported C++ language standards to use for compiling C++ sources in the package.
+The supported C++ language standard to use for compiling C++ sources in the package.
```

Comment:
Even if an `enum` will result on 1 standard, I would keep `standards` because below we are having a list of standard.